astro/src/content/docs/get-started/core-concepts/integration-points.mdx

```diff
@@ -69,30 +70,36 @@ Each JWT has a header, a payload and a signature. Here's a diagram of a JWT:
 
 ### JWT Storage
 
-FusionAuth recommends the Authorization Code grant, where there is a server side component which exchanges the one-time use authorization code for an access token. This server side component offers a lot of flexibility when it comes to storing the JWT.
+FusionAuth recommends the Authorization Code grant, where there is a server-side component which exchanges the one-time use authorization code for an access token. This server side component offers a lot of flexibility when it comes to storing the JWT.
 
 But what should you do with it? After all, an access token is a bearer token, and should be properly secured. What you do with the token depends on what you are using it for as well as your security requirements. If you are wondering about your options, [here are a number of common login flows](/learn/expert-advice/authentication/login-authentication-workflows).
 
 Recommended storage mechanisms include:
 
 * sending the token down to the browser as a `Secure`, `HttpOnly` cookie. If you don't require cross-site cookie sharing, set `SameSite` to `Strict`. Otherwise, set `SameSite` to `Lax`, which will share the cookies in certain situations. Learn more about [`SameSite` settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).
```

Comment:
```suggestion
* **_Cookie_** - sending the token down to the browser as a `Secure`, `HttpOnly` cookie. If you don't require cross-site cookie sharing, set `SameSite` to `Strict`. Otherwise, set `SameSite` to `Lax`, which will share the cookies in certain situations. Learn more about [`SameSite` settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).
```

Reply:
good suggestion to have these line up with the table headers.